src/main/java/edu/ucsb/cs156/rec/controllers/RecommendationRequestController.java

```diff
@@ -131,6 +131,7 @@ public RecommendationRequest updateRecommendationRequestAsAdmin(
                 .orElseThrow(() -> new EntityNotFoundException(RecommendationRequest.class, id));
 
         recommendationRequest.setStatus(incoming.getStatus());
+        recommendationRequest.setCompletionDate(LocalDateTime.now());
```

Comment:
I wonder if we should check the status, and only set this if the status has been set to "COMPLETED" or "DENIED".

Also: is it possible for the instructor change this from "COMPLETED" or "DENIED" back to "PENDING", and in that case, should the completion date be set back to null?